functionalDB/chapter.md

```diff
@@ -1,103 +1,104 @@
-# Designing a database like an archaeologist
+<!-- American spelling --> 
+# CircleDB: An Archaeology-Inspired Database

```

Comment:
If it is a real estate issue, I think it is better to remove the CircleDB out, as this conveys no information to the reader about what's in the chapter or why to read it.